ui/cypress/integration/e2e/Solutions/steps.js

```diff
@@ -16,7 +16,7 @@ When(
       '/api/kubernetes/api/v1/namespaces?labelSelector=solutions.metalk8s.scality.com/environment',
     ).as('getEnvironmentsList');
 
-    cy.get('.sc-sidebar-item').eq(2).click(); // go to solutions list
+    cy.get('.sc-sidebar-item').eq(3).click(); // go to solutions list
```

Comment:
That's why we should provide `data-cy` attributes on all items (cc @ChengYanJin :stuck_out_tongue_winking_eye:)